### PR TITLE
Quotesy

### PR DESCRIPTION
**Pull request by natesilva**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/quotesy/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * @param tag the tag to search for
+ */
+export function random_by_tag(tag: string): Quote;
+
+/** A single quote from the database */
+export interface Quote {
+    /** The author of the quote */
+    author: string;
+    /** The text of the quote */
+    text: string;
```
Comment (nelsonic) [inline types/quotesy/index.d.ts:27]: Suggest adding `tags` and `source` (optional keys):
```suggestion
    text: string;
    /** Tags; words associated with the quote **/
    tags: string;
    /** Source; a URL where the origin of the quote can be verified **/
    
```

**Review by nelsonic — APPROVED** [review]
@natesilva LGTM. 👍 
Could be worth adding `source` and `tags` to the interface, but otherwise all good.

**Review by martin-badin — APPROVED** [review]

**Comment by natesilva** [comment]
Merged suggested changes by @nelsonic.

**Review by nelsonic — APPROVED** [review]
Nice additions @natesilva 👍

**Pull request merged**